fhe-toolkit-macos.xcodeproj/project.pbxproj

```diff
@@ -758,15 +1204,15 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_CXX_LANGUAGE_STANDARD = "c++14";
-				CODE_SIGN_ENTITLEMENTS = "fhe-toolkit-macos/fhe-toolkit-macos.entitlements";
+				CODE_SIGN_ENTITLEMENTS = "Privacy Preserving Search/fhe-toolkit-macos.entitlements";
 				CODE_SIGN_IDENTITY = "-";
 				CODE_SIGN_STYLE = Automatic;
 				COMBINE_HIDPI_IMAGES = YES;
 				DEVELOPMENT_TEAM = 76CMF53Q35;
 				ENABLE_HARDENED_RUNTIME = YES;
```

Comment:
`ENABLE_HARDENED_RUNTIME = YES; `
I thought we had changed this to `NO` to avoid error below.
error: Ad Hoc code signing is not compatible with the Hardened Runtime. Sign with a Mac Developer certificate or disable the Hardened Runtime by setting the ENABLE_HARDENED_RUNTIME build setting to NO. (in target 'Privacy Preserving Search')

Reply:
changes were made, thanks for pointing this out